src/sage/combinat/integer_vector.py

```diff
@@ -780,6 +780,37 @@ def __contains__(self, x):
                 return False
         return True
 
+    def _unrank_helper(self, x, rtn):
+        """
+        return an element at rank ``x``.
```

Comment:
`Return the` instead of `return an`.

I also forgot:
```
INPUT:

    - ``x`` - a nonnegative integer
    - ``rtn`` - a list of nonnegative integers

Return an element of rank ``x`` by iterating through all integer vectors beginning with ``rtn``.
```